com.unity.render-pipelines.high-definition/Runtime/Utilities/DeviceInfo.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    public class DeviceInfo
```

Comment:
Same as above, there is currently no point in making this public so I think we should make it internal.